### PR TITLE
Test Vector Generation works again

### LINKED ISSUE #34: TypeError: Cannot read property 'uncompressed' of undefined
https://github.com/keybase/warpwallet/blob/dfc3ea3ff88f4246146f903a0c37c40bcc650fa9/test/gen/gen_spec.iced#L114
'make spec' stuck here saying:
TypeError: Cannot read property 'uncompressed' of undefined

### PR DESCRIPTION
**Pull request by gallizoltan**
bu (bitcoin utils) has been renamed to ku (key utility) and its syntax changed
fixes #34

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ it does use npm to manage its dependencies
 ## Build
 
 ```sh
-$ npm install -g iced-coffee-script
+$ sudo ln -s /usr/bin/nodejs /usr/bin/node
```
Comment (maxtaco) [inline README.md:15]: Can you remove these changes?  I'm assuming you've installed node as your user and it's in your path.

File: README.md [hunk 2]
```diff
@@ -27,6 +28,8 @@ To generate our reference test vectors, we use the reference Scrypt implementati
 Python PBKDF2, and a Python library to turn a seed into a keypair.  To see how this works, try:
 
 ```sh
+$ pip install virtualenv
+$ sudo /usr/bin/easy_install virtualenv
```
Comment (maxtaco) [inline README.md:34]: is this line needed?

File: test/gen/Makefile [hunk 3]
```diff
@@ -1,8 +1,9 @@
 
+SHELL := /bin/bash
```
Comment (maxtaco) [inline test/gen/Makefile:2]: why is this needed?  I use `sh`

**Review by maxtaco — COMMENTED** [review]
Thanks for the PR, I'll try it out.

**Comment by gallizoltan** [comment]
Sorry for these commits, I was not aware GitHub automatically adds new commits to my pull request. I will close this PR and make a new one, focusing only to make Test Vector Generation work.

File: README.md (hunk 1 above)
Comment (gallizoltan) [reply README.md:15]: Ok, I will remove this.

File: README.md (hunk 2 above)
Comment (gallizoltan) [reply README.md:34]: This is needed only at the first run.

File: test/gen/Makefile (hunk 3 above)
Comment (gallizoltan) [reply test/gen/Makefile:2]: Without this the `make build` fails with this message:
```
/bin/sh: 1: source: not found
Makefile:15: recipe for target 'build-python' failed
make: *** [build-python] Error 127
```
I googled for "/bin/sh: 1: source: not found" and found this solution:
https://stackoverflow.com/questions/670191/getting-a-source-not-found-error-when-using-source-in-a-bash-script

I don'n know, I have a Ubuntu 16.04 LTS. What environment are you using for `sh` to find `source`?

**Comment by gallizoltan** [comment]
Let's try it again: https://github.com/keybase/warpwallet/pull/36
